includes/classes/shopping_cart.php

```diff
@@ -611,19 +611,20 @@ function remove_all() {
   }
   /**
    * Method return a comma separated list of all products in the cart
+   * NOTE: Not used in core ZC, but some plugins make use of it as a helper function
    *
    * @return string
-   * @todo ICW - is this actually used anywhere?
```

Comment:
Just a heads up: I actually have shipping modules that use this function so while it appears un-needed it is usefully for obtaining the string of products_id in the cart.